NestedAccordion/js/scripts.js

```diff
@@ -0,0 +1,45 @@
+/*
+What does it do?
+  What interactive elements exist?
+    * headings- when you click on the headings they open accordion style and then
+      there are accordions with in accordions
+    * navagation buttons- `::before` & `::after` js???
+  How do you interact with them?
+  What visual effects are produced by interacting?
+
+How does it do it?
+ What existing HTML elements are changed?
+ What new HTML elements are created?
+ What CSS styles are used to produce the effect?
+  * `cursor: pointer` controls animated arrow?
+  *
+  **/
+
+The HTML language is assigned to english and the class should be assigned to Javascript.
```

Comment:
like we saw in class, you can comment out your pseudocode and break the problem down into smaller steps.